packages/common/src/services/dictionaryService.ts

```diff
@@ -0,0 +1,30 @@
+import { isEmpty } from 'lodash-es';
+import { Dependencies } from '../config/config.js';
+
+import getCategoryUtils from '../utils/categoryUtils.js';
+import getDictionaryUtils from '../utils/dictionaryUtils.js';
+
+const dictionaryService = (dependencies: Dependencies) => {
+	return {
+		register: async (categoryName: string, dictionaryName: string, version: string) => {
+			const { createCategoryIfDoesNotExist } = getCategoryUtils(dependencies);
+			const savedCategory = await createCategoryIfDoesNotExist(categoryName);
+			if (isEmpty(savedCategory)) return;
```

Comment:
Can `createCategoryIfDoesNotExist` even return an empty result? A quick read of the code shows it either throws an error or returns a value...

If it does return an empty value, should we at least log this issue before returning our undefined response?